### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by busy-work-only** [dotnet/dotnet-operator-sdk#796] [Q&A]
The `KubeOps.Templates` package seems to set up a new project that references the old (deprecated) `KubeOps` package. Is the intention that we just try to follow the examples in the repo, or is there a newer (undocumented) template to use?

**Comment by buehler** [comment]
Hey @busy-work-only 

Yes, the templates package is currently deprecated. I didn't find the time to update the templates for the new versions. However, the examples in the repository show how to create the operator.

Also, the documentation should be quite helpful.

**Reply by busy-work-only** [reply]
I thought that was the case but NuGet doesn't indicate as such like some of the other old packages.

Where is the documentation? The [docs folder](https://github.com/buehler/dotnet-operator-sdk/tree/0965d2317fe4dab422893a76ff0a22297bb9dfae/docs) is effectively empty.
